website/content/en/docs/concepts/nodepools.md

```diff
@@ -157,7 +157,7 @@ spec:
       duration: 8h
       nodes: "0"
 
-  # Resource limits constrain the total size of the cluster.
+  # Resource limits constrain the total size of the pool.
```

Comment:
Nice catch! Do you mind applying this change to all docs versions in the website folder?

Reply:
Oops, didn't notice the other versions. Done 👍